test/test_pipelines.py

```diff
@@ -2,6 +2,7 @@
 import transformers
 
 import gradio as gr
+from gradio.interface import Interface
```

Comment:
Everything else in this file is using `gr.Interface`; the new tests should probably too.